packages/shared/src/HeaderButton.js

```diff
@@ -10,16 +10,14 @@ type Props = {|
   +onPress: () => void,
   +onLongPress?: (React.ElementRef<typeof Touchable>) => void,
   +children?: React.Node,
+  +left?: boolean,
 |};
 
 type State = {|
   reference: React.ElementRef<typeof Touchable> | null,
 |};
 
-export default class HeaderRightButton extends React.PureComponent<
-  Props,
-  State,
-> {
+export default class HeaderButton extends React.PureComponent<Props, State> {
```

Comment:
This component is a good candidate now for testing (even if Snapshots) for `.Left` and `.Right`.

Reply:
Done.